.dockerignore

```diff
@@ -1,5 +1,6 @@
 node_modules
+package-lock.json
```

Comment:
If we update `package-lock.json` on Windows and then builds on Linux it pulls the wrong GSSAPI pacakge.